CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ Beta 0.5.0
 ----------
 Added functionality:
 * Added `nanopore_assemblies` as an alias for directory uploads
+* Added support for uploading `fast5` files
+* Added `--upload_mode=<mode>` as an argument, replacing the `--assemblies` argument
```

Comment:
Still need this information added to the main docs that get published on readthedocs

Reply:
Added.

Reply:
I still see a `--assemblies` section in https://irida-uploader.readthedocs.io/en/stable/index.html#assemblies-uploading

Reply:
Oops, committing too fast, fixed now